spec/factories/my_module_status_conditions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
We do not need this factory, this is a superclass and should not be saved.

Reply:
We can implement factory for specific conditions/consequences or implications. Like `read_only.rb`